notebooks/information_extraction_raven.ipynb

```diff
@@ -0,0 +1,2726 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.    User Query: Save data from the provided text. START TEXT:{{docs[0].content|replace("\n"," ")|truncate(10000)}} END TEXT
</pre><p>Do you mean "extracting data" or "saving data"?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepset-ai/haystack-cookbook/pull/67/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
Empirical.

I think it makes more sense for the LLM to create a function call with the extracted data and pass it for saving.